src/browser/new-browser.ts

```diff
@@ -77,13 +83,17 @@ export class NewBrowser extends Browser {
         try {
             this.setHttpTimeout(this._config.sessionQuitTimeout);
             await this._session!.deleteSession();
+            this._wdProcess?.free();
         } catch (e) {
             warn(`WARNING: Can not close session: ${(e as Error).message}`);
+            this._wdProcess?.kill();
```

Comment:
If session can't be closed, we kill webdriver process, so later we could launch the new one